.github/workflows/wait_for_required_workflows.yml

```
@@ -27,6 +29,9 @@ jobs:
               // This is a GitHub limitation https://github.com/cloudquery/cloudquery/issues/2688
               throw new Error('Too many files changed. Skipping check. Please split your PR into multiple ones.')
             }
+
+            const can_skip = process.env.CAN_SKIP.split("\n")
```

```suggestion
            const canSkip = ['validate-release / validate-release']
```

Not sure why we need the env variable and use the casing used in JS (and in this script)